tools/uniffi-bindgen-library-mode/src/main.rs

```diff
@@ -127,7 +127,6 @@ fn run_uniffi_bindgen(cli: Cli) -> Result<()> {
                 module_name: Some(module_name),
                 modulemap_filename,
                 metadata_no_deps: false,
-                link_frameworks: vec![],
```

Comment:
ouch, we really should have derived `Default` for that struct?

Reply:
Yeah, that's a great point.